### PR TITLE
Gun rebalance, Mostly DMR

### PR DESCRIPTION
**Pull request by David-Stormwell**
## About The Pull Request

Buffs most of the T Series wielding speed, and gives the DMR a few buffs such as incendiary only req ammo, increased ammo capacity for normal magazines, it can now use a gyro and 5 less damage for normal ammo.
Also adds the DMR and DMR ammo to req vendors and fixes the rail attachment offset a little more. 
incendiary DMR ammo does 35/15
Also some Survivor guns have more attachments and the custom shotgun's burst delay from 6 to 5.

## Why It's Good For The Game

More marine buffs and toys to play with. The DMR was still a heavily underused gun due to the ammo problem, and for a greater reason to use it incendiary ammo was given. Also RenGusta and a few others requested it and the wield delay buffs.

## Changelog
:cl:
add: Added DMR incendiary ammo and tweaked the DMR's rail attachment offsets
balance: Buffed most T Series wield delays
balance: More Survivor guns have more raillight additions to slap on
imageadd: Incendiary DMR magazine 
/:cl:

### CODE REVIEW

**Comment by David-Stormwell** [comment]
If someone could explain why travis is angry that would be neat, else I'll close and restart the whole thing later

**Comment by psykzz** [comment]
Travis couldn't build tgui, which is unrelated.

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -449,17 +449,25 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	penetration = 20
 
 /datum/ammo/bullet/rifle/standard_dmr
-	name = "high velocity rifle bullet"
+	name = "marksmen bullet"
```
Comment (Rohesie) [inline code/modules/projectiles/ammo_datums.dm]:
```suggestion
	name = "marksman bullet"
```

File: code/modules/projectiles/guns/rifles.dm [hunk 2]
```diff
@@ -142,8 +142,8 @@
 	unload_sound = 'sound/weapons/guns/interact/m41a_unload.ogg'
 	reload_sound = 'sound/weapons/guns/interact/m41a_reload.ogg'
 	caliber = "10x27mm caseless" //codex
-	aim_slowdown = 0.75
-	wield_delay = 1.0 SECONDS
+	aim_slowdown = 0.5
```
Comment (Rohesie) [inline code/modules/projectiles/guns/rifles.dm]:
```suggestion
	aim_slowdown = 0.05 SECONDS
```

File: code/modules/projectiles/guns/shotguns.dm [hunk 3]
```diff
@@ -203,7 +204,7 @@ can cause issues with ammo types getting mixed up during the burst.
 
 	fire_delay = 10
 	burst_amount = 2
-	burst_delay = 6
+	burst_delay = 5
```
Comment (Rohesie) [inline code/modules/projectiles/guns/shotguns.dm]:
```suggestion
	burst_delay = 0.5 SECONDS
```

**Comment by Rohesie** [comment]
I don't like the fact that these are all straight marine buffs, but let's test it.

**Pull request merged**